xml/System.Net.Http/WinHttpHandler.xml

```diff
@@ -392,8 +392,8 @@
         <Parameter Name="disposing" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="disposing">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="disposing"><see langword="true" /> to release both managed and unmanaged resources; <see langword="false" /> to releases only unmanaged resources.</param>
```

Comment:
```suggestion
        <param name="disposing"><see langword="true" /> to release both managed and unmanaged resources; <see langword="false" /> to release only unmanaged resources.</param>
```

Reply:
Looks like copy-paste error from HttpClientHandler - can you fix it there as well please?